.env

```diff
@@ -9,7 +9,9 @@ SENTRY_IMAGE=getsentry/sentry:nightly
 SNUBA_IMAGE=getsentry/snuba:nightly
 RELAY_IMAGE=getsentry/relay:nightly
 SYMBOLICATOR_IMAGE=getsentry/symbolicator:nightly
+VROOM_IMAGE=getsentry/vroom:nightly
 WAL2JSON_VERSION=latest
 HEALTHCHECK_INTERVAL=30s
 HEALTHCHECK_TIMEOUT=60s
 HEALTHCHECK_RETRIES=10
+SENTRY_BUCKET_PROFILES=file://localhost//var/lib/sentry-profiles
```

Comment:
Why is this an env variable and not in a config file?

Reply:
It's because.. uhhh... https://github.com/getsentry/vroom/blob/bdda64cc531de12ff86fc2a547850712a231ceeb/cmd/vroom/config.go#L4-L19

Reply:
(Git blamed to my commit lol)

Reply:
@BYK I put this here as I understood that's where you put environment variables users are susceptible to edit for their needs. Now, this one is a bit special because if you wanted to put a different storage location (like a bucket on GCS or S3), you'd have to pass additional environment variables and thus modify the `docker-compose.yml` file anyway.

Is this usually how you are doing this to customize the behavior of some services?

Reply:
@phacops not really unless the setting applies to multiple services (like retention days). The typical way is to have a `<service_name>` folder with an example config we provide for defaults and then folks can customize it to their needs. See https://github.com/getsentry/self-hosted/tree/master/sentry or https://github.com/getsentry/self-hosted/tree/master/relay for examples.